third_party/Makefile

```diff
@@ -5,11 +5,16 @@
 
```

Comment:
Shall we separate the PR?
One is only for `Benchmarks: Build Pipeline - Support rocm in third_party/makefile` and the other is `Benchmarks: Build Pipeline - Add rccl-tests as a submodule with building logic`

Reply:
sure